frontend/src/pages/pipelines/global/modelCustomization/utils.ts

```diff
@@ -163,3 +171,40 @@ export const translateIlabFormToBaseModelInput = (
   sdg_base_model: data.baseModel.sdgBaseModel,
   // TODO more output model fields
 });
+
+export const translateIlabFormToRunType = (
+  data: ModelCustomizationFormData,
+): {
+  runType: string;
+} => ({
+  runType: data.runType.value,
+});
+
+export const translateIlabFormToHyperparameters = (
+  data: ModelCustomizationFormData,
+): {
+  hyperparameters: Record<string, RuntimeConfigParamValue | undefined>;
+} => ({
+  hyperparameters: data.hyperparameters,
```

Comment:
its not under `hyperparameters `

just spread it

```suggestion
  ...data.hyperparameters,
```